src/ResponseMerger/NuxtResponseMerger.php

```diff
@@ -73,7 +73,7 @@ public function mergeResponses(ResponseInterface $backendResponse, ResponseInter
         }
       }
       if ($metatags_html != '') {
-        $page = preg_replace("/<head(.*?)>/", '<head$1>' . $metatags_html, $page);
+        $page = preg_replace("/<head(.*?)>/", '<head$1>' . $metatags_html, $page, 1);
```

Comment:
Why can the head tag appear more often and why do we need to treat the first one only? This deservers a comment.

Reply:
:/ html tags got removed from previous comment,
the issue was that this regex matches `head` as well as `header` tags, header tags are in the body, and metadata would be added multiple times, the limit will solve this as `head` always comes before body.

Reply:
ok, got it. That's a good find - but as this is not obvious from the code, the more important is the comment. As said:
>This deservers a comment.
Obviously and as always: in code.

Reply:
Documented in code